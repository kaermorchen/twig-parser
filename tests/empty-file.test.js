import { test, expect } from 'vitest';
import Parser from '../out/parser.js';

test('Empty twig', () => {
  const twig = `43`;

  expect(Parser.parse(twig)).toMatchSnapshot();
});
