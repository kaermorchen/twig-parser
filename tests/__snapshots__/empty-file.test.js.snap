// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Empty twig 1`] = `"q"`;
